lib/cmorizer.rb

```diff
@@ -15,10 +15,30 @@ def initialize(src_txt)
       @experiments = []
       @years_step = 1
       @eval_mode = true
+      src_txt = sanitize src_txt
       instance_eval(src_txt, src_txt)
       @eval_mode = false
     end
     
+    def sanitize(src_txt)
+  # Sanitizes the source text to avoid mistreating variable names as Ruby constants
+      new_txt = []
+      requires_sanitization = false
+      src_txt.each_line do |line|
+        if line.start_with?("cmorize ") && line.include?("=>")
+          first, varname, *rest = line.split
+          if varname[0] =~ /[A-Z]/
+            requires_sanitization = true
+            new_txt << "#{varname} = '#{varname}'"
+          end
+        end
+      new_txt << line
+      end
+      if requires_sanitization
+        src_txt = new_txt.join("\n")
+      end
+      src_txt
+      end
```

Comment:
Is the indentation correct in this case?

Reply:
Yep, indentation needs to be addressed here.

Reply:
@siligam: If you aren't aware of this feature yet; GitHub reviewing offers "change suggestions" which you can do by pushing the little button with a page and + - symbols, or directly in markdown with a code-fenced tagged with "suggestion" and then using `diff` style syntax with plus and minus. As a fence it looks like this:

````
```suggestion
+ add this line
- remove this one
```
````

Docs are [here under step \#7](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request).

If you already knew that, emm...sorry for the spam :)